### ISSUE TITLE
[OTHER]: Clarification on nanobind vs. pybind11

### ISSUE THREAD

**Issue by blais** [wjakob/nanobind#404]
### Please don't use.

Hi Wenzel!
I just discovered nanobind, and I'd been a fan of pybind11, which if I'm not mistaken is also something you wrote.
Can you provide some insights (ideally on the main page) for its replacement, and whether you'd recommend using nanobind over pybind11 in all cases? Was is a full rewrite? Are there cases where pybind11 would still be what you'd recommend?
That would provide useful context for users. Thank you!

**Comment by blais** [comment]
Ah! Actually, you discuss it in Chapter Two of the documentation.
Thanks for writing that.
It would probably save people time to add a little something of that on the front page regardless.
I'll close this for now.
Thanks!

**Comment by wjakob** [comment]
TLDR: It's a tiny rewrite where I tried to fix some things that bothered me. pybind11 is too complex and has too many downstream users to make those kinds of radical changes.

There are some things that pybind11 could do that are no longer possible with nanobind. If your project doesn't use those features, then I would strongly recommend that you use nanobind instead of pybind11.
